src/pudl/transform/classes.py

```diff
@@ -782,7 +852,14 @@ class AbstractTableTransformer(ABC):
     """Cached intermediate dataframes for use in development and debugging."""
 
     parameter_model = TableTransformParams
+    """The :mod:`pydantic` model that is used to contain & instantiate parameters.
+
+    In child classes this should be replaced with the data source-specific
+    :class:`TableTransformParams` class, if it has been defined.
+    """
+
     params: parameter_model
+    """The parameters that will be used to control the transformation functions."""
```

Comment:
wait this is from a previous pr but why is `parameter_model` defined above and then `params` is defined? is this a magic pydantic typing thing? some explanation would be helpful

Reply:
`params` is an instance of type `parameter_model` (the Pydantic model which is being used to define the parameters). This varies across the different `AbstractTableTransformer` subclasses, and it needs to be known explicitly for validation & instantiation of the parameters.

Reply:
oh okay. The changes are definitely helpful. An additional addition for me would be to note in the definition of `parameter_model` that this is being defined explicitly for validation  & instantiation of the parameters

Reply:
I've expanded the explanation and added a pointer to examples from the Ferc1 table transformers.